City-of-Helsinki/spouse-program
# Fixing Composer

Composer install resulted in some clashes with modules stored in Git.

Added Preserve-paths to composer and set composer up in a way that it will not replace wanted modules or their versions. Also removed some language files, since there is no use for them. 

Testing guide:
- Do `composer install`
- Check that git has no alterations with `git status`
- Install the rest of the site and see that everything works.
